packages/hawtio/src/plugins/springboot/index.ts

```diff
@@ -0,0 +1,18 @@
+import { hawtio, HawtioPlugin } from '@hawtiosrc/core'
+
+import { pluginId, pluginPath } from './globals'
+import { workspace } from '@hawtiosrc/plugins'
+import { helpRegistry } from '@hawtiosrc/help'
+import help from './help.md'
+import {Springboot} from "@hawtiosrc/plugins/springboot/Springboot"
+
+export const springboot: HawtioPlugin = () => {
+  hawtio.addPlugin({
+    id: pluginId,
+    title: 'Springboot',
+    path: pluginPath,
+    component: Springboot,
+    isActive: async () => workspace.hasMBeans(),
```

Comment:
This condition doesn't seem correct. Check the original code.
https://github.com/hawtio/hawtio-integration/blob/deebfd1ea93bd6e0cfa5bf63b9d7e7768a8b6110/plugins/spring-boot/spring-boot.service.ts#L11-L22